Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ReviewRequestController.java

```diff
@@ -33,18 +34,24 @@
         return reviewRequest;
     }
     
-    @RequestMapping(value = "/auth/request/resolve/{reviewId}", method = RequestMethod.POST )
+    @PostMapping(value = "/auth/request/resolve/{reviewId}")
     public @ResponseBody ReviewRequest resolveRequest(@PathVariable String reviewId,@RequestBody Comment comment,Principal principal) {
         return requestsMgr.updateReview(reviewId, ReviewStatus.RESOLVED, comment, OffsetDateTime.now(), principal.getName());
     }
     
-    @RequestMapping(value = "/auth/request/reopen/{reviewId}", method = RequestMethod.POST )
+    @PostMapping(value = "/auth/request/reopen/{reviewId}")
     public @ResponseBody ReviewRequest reopenRequest(@PathVariable String reviewId,@RequestBody Comment comment,Principal principal) {
         return requestsMgr.updateReview(reviewId, ReviewStatus.OPENED, comment, OffsetDateTime.now(), principal.getName());
     }
     
-    @RequestMapping(value = "/auth/request/{conceptId}/all", method = RequestMethod.GET)
+    @GetMapping(value = "/auth/request/{conceptId}/all")
     public @ResponseBody List<ReviewRequest> getAllReviews(@PathVariable String conceptId, Principal principal){
         return requestsMgr.getAllReviews(conceptId);  
     }
+    
+    @GetMapping(value="/auth/openrequests")
```

Comment:
should follow the other url patterns, e.g. `/auth/request/open` or `/auth/request/all/open`